index.js

```diff
@@ -26,4 +26,22 @@ isStream.transform = stream =>
 	typeof stream._transform === 'function' &&
 	typeof stream._transformState === 'object';
 
+isStream.type = stream => {
+	let type;
+
+	if (isStream.transform(stream)) {
+		type = 'transform';
```

Comment:
This and the others can just return directly. Then you don't need the `else` either.

Reply:
Done 6f6dfee3cb29ef9e2262b6ba93bd4f9cfb0e260f